packages/@ngtools/webpack/src/transformers/replace_resources.ts

```diff
@@ -131,7 +131,10 @@ function _getContentOfKeyLiteral(node?: ts.Node): string | null {
 }
 
 function _getResourceRequest(element: ts.Expression, sourceFile: ts.SourceFile) {
-  if (element.kind == ts.SyntaxKind.StringLiteral) {
+  if (
+    element.kind === ts.SyntaxKind.StringLiteral ||
+    element.kind === ts.SyntaxKind.FirstTemplateToken
```

Comment:
This is wrong, it should be `ts.SyntaxKind.NoSubstitutionTemplateLiteral`. You're not getting the last position of the template token correctly in this case. We probably don't want substitutions in the URLs.

Reply:
My mistake, I have made the relevant changes @hansl